t/FirstRoboticsTeams.t

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use DDG::Test::Spice;
+
+spice is_cached => 1;
+
+ddg_spice_test(
+    [qw( DDG::Spice::FirstRoboticsTeams)],
+    # At a minimum, be sure to include tests for all:
+    # - primary_example_queries
+    # - secondary_example_queries
```

Comment:
You can remove these comments :)

Also you could write a `build_test` function to insert the team name and get rid of the repetitive tests. [This](https://github.com/GuiltyDolphin/zeroclickinfo-goodies/blob/port-timer/t/Timer.t#L12-L33) is a similar example.